src/tests/orca-jedi/test_interpolator.cc

```diff
@@ -121,8 +121,8 @@ CASE("test basic interpolator") {
 
     double missing_value = util::missingValue<double>();
     std::vector<double> testvals = {
-      18.488888892, missing_value, 18.1592999503,
-      17.9419381132, missing_value, 17.75000288,
+      18.4888916016, missing_value, 18.1592999503,
```

Comment:
I was wondering why these numbers have changed?

Reply:
this is because the numbers we compare against are stored in the program as floats now by default. This gives a small change in the results of the interpolation, as it is now applied on float fields instead of double atlas fields.